forem/forem
# Remove bootstrap from admin/badge_achievements, admin/organizations and admin/podcasts

## What type of PR is this? (check all applicable)


## Description

This PR is a step towards removing bootstrap completely and replacing it with crayons.

The changes are only design related for which all before vs after screenshots have been shown below.

## Related Tickets & Documents



- Related Issue #18756
- Closes #

## QA Instructions, Screenshots, Recordings

### Badge Achievements
**How to test this?**
1. Sign in as admin locally
2. Visit `/admin/content_manager/badge_achievements`
3. Visit the above page on main branch and on this PR's branch to see Ui differences.
4. Mainly notice that the cover of warning has changed as we are now using crayons classes i.e. `crayons_notices`

| Before  | After |
| ------------- | ------------- |
| <img width="1512" alt="Screenshot 2022-12-16 at 1 46 57 PM" src="https://user-images.githubusercontent.com/9396084/208153781-4a902b4e-386a-480d-a076-70321a762e65.png"> | <img width="1512" alt="Screenshot 2022-12-16 at 1 49 37 PM" src="https://user-images.githubusercontent.com/9396084/208153798-782184f3-1013-43cc-bc3e-1ac27f700749.png"> |

### Organizations
**How to test this?**
1. Sign in as admin locally
2. Visit `/admin/content_manager/organizations`
3. Visit the above page on main branch and on this PR's branch to see Ui differences.

| Before  | After |
| ------------- | ------------- |
| <img width="1512" alt="Screenshot 2022-12-16 at 10 48 21 PM" src="https://user-images.githubusercontent.com/9396084/208154174-5676bece-de14-45b4-bc93-71fda4e9ae06.png"> | <img width="1512" alt="Screenshot 2022-12-16 at 10 46 44 PM" src="https://user-images.githubusercontent.com/9396084/208154166-01658910-6008-4b02-b8da-1e4edab0c39d.png"> |
| <img width="1512" alt="Screenshot 2022-12-16 at 10 48 28 PM" src="https://user-images.githubusercontent.com/9396084/208154179-094bf14a-ba3f-495d-ad42-ce5cdd42ba94.png"> | <img width="1512" alt="Screenshot 2022-12-16 at 10 46 56 PM" src="https://user-images.githubusercontent.com/9396084/208154172-fcee9a40-c03f-4ba2-b073-1d47d57383eb.png"> |

### Podcasts
**How to test this?**
1. Sign in as admin locally
2. Visit `/admin/content_manager/podcasts`
3. Visit the above page on main branch and on this PR's branch to see Ui differences.
4. Notice that the main difference is in search-box only.

| Before  | After |
| ------------- | ------------- |
| <img width="1512" alt="Screenshot 2022-12-16 at 11 10 39 PM" src="https://user-images.githubusercontent.com/9396084/208157472-b2d4ae18-4f9a-4122-b075-f7b075523aba.png"> | <img width="1512" alt="Screenshot 2022-12-16 at 11 10 25 PM" src="https://user-images.githubusercontent.com/9396084/208157443-c974212c-b7b0-4e21-a959-fdeae9598c68.png"> |

### UI accessibility concerns?

No

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_
